src/Microsoft.Azure.WebJobs.Protocols/PersistentQueueMessage.cs

```diff
@@ -25,7 +24,7 @@ internal abstract class PersistentQueueMessage
 
         /// <summary>Gets or sets the blob object.</summary>
         [JsonIgnore]
-        internal ICloudBlob Blob { get; set; }
+        internal object Blob { get; set; } // It's a ICloudBlob, but use System.Object to break coupling. 
```

Comment:
I don't think we should change this - there's got to be a better way to accomplish your goal. What's the problem?

Reply:
@MikeStall can you comment?